packages/notifi-react/lib/context/NotifiTargetContext.tsx

```diff
@@ -80,18 +89,54 @@ const formatTelegramForSubscription = (telegramId: string) => {
 export type UpdateTargetInputs = <T extends 'form' | 'toggle'>(
   target: T extends 'form'
     ? Extract<Target, 'email' | 'phoneNumber' | 'telegram'>
-    : Extract<Target, 'discord' | 'slack'>,
-  value: T extends 'form' ? TargetInputFromValue : boolean,
+    : Extract<Target, 'discord' | 'slack' | 'wallet'>,
+  value: T extends 'form' ? { value: string; error?: string } : boolean,
```

Comment:
I guess using `TargetInputFromValue` instead of inline type definition is better